projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -386,6 +386,29 @@ protected function __construct() {
 		wp_register_style( 'grunion.css', GRUNION_PLUGIN_URL . 'css/grunion.css', array(), JETPACK__VERSION );
 		wp_style_add_data( 'grunion.css', 'rtl', 'replace' );
 
+		wp_enqueue_style(
+			'jquery-ui-css',
+			'https://code.jquery.com/ui/1.13.2/themes/base/jquery-ui.css',
+			array(),
+			JETPACK__VERSION
+		);
+
+		wp_enqueue_script(
+			'jquery-ui',
+			'https://code.jquery.com/ui/1.13.2/jquery-ui.js',
+			array( 'jquery' ),
+			JETPACK__VERSION,
+			true
+		);
+
+		wp_enqueue_script(
+			'jetpack-dropdown-test',
+			plugins_url( 'js/form-style.js', __FILE__ ),
+			array( 'jquery', 'jquery-ui' ),
+			JETPACK__VERSION,
+			true
+		);
+
```

Comment:
Right now, my main question is about the best way to handle those dependencies.
Is this file the correct place to do it?
Also, enqueuing scripts and styles through its URL don't feel much right.

Reply:
Yeah, I think that's the right place to do it. You might want to add a comment these are dependencies for the dropdown block or something. Or move this and the two lines above into a new method that's responsible for enqueuing style and js dependencies for blocks to make it self explanatory.